src/editors/EditorContainer.tsx

```diff
@@ -50,7 +60,21 @@ export default function EditorContainer<R, SR>({
   }, [getInputNode]);
 
   // close editor when scrolling
-  useEffect(() => onCommitCancel, [scrollTop, scrollLeft, onCommitCancel]);
+  useEffect(() => {
+    if (scrollLeft !== prevScrollLeft.current || scrollTop !== prevScrollTop.current) {
+      commitCancel();
+    }
+  }, [commitCancel, scrollLeft, scrollTop]);
```

Comment:
Why not keep it like that?
```suggestion
  useEffect(() => commitCancel, [scrollTop, scrollLeft, commitCancel]);
```

Reply:
It fires when the component is unmounted. We want to call `commitCancel` only when the scroll position is changed